src/main/java/lotto/domain/LottoMatchInformation.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+public enum LottoMatchInformation {
+    THREE(3, 0, 5000),
```

Comment:
```suggestion
    THREE(3, 0, 5_000),
```
이런식으로 가독성을 높혀줄 수 있습니다 ㅎㅎ